netbox/netbox/views/generic/bulk_views.py

```diff
@@ -541,6 +541,13 @@ class BulkEditView(GetReturnURLMixin, BaseMultiObjectView):
     def get_required_permission(self):
         return get_permission_for_model(self.queryset.model, 'change')
 
+    def extra_object_field_operations(self, form, obj):
```

Comment:
I'd suggest renaming this to something like `post_save_operations()`, and moving the M2M and add/remove tag logic that currently lived near the bottom of `_update_objects()` here. That way all the post-save behavior becomes completely accessible.